packages/calcite-components/src/components/block-section/block-section.scss

```diff
@@ -1,3 +1,14 @@
+/**
```

Comment:
We might want to hold off on documenting these. The review from https://github.com/Esri/calcite-design-system/pull/8594 might affect some props. If we want to be extra careful, we might need to rename these as internal.

Reply:
Sure, that makes sense.